ESP8266Interface.h

```diff
@@ -18,8 +18,18 @@
 #define ESP8266_INTERFACE_H
 
 #include "mbed.h"
-#include "ESP8266.h"
 
+#include "events/EventQueue.h"
+#include "events/mbed_shared_queues.h"
+#include "rtos/Thread.h"
```

Comment:
`rtos/Thread.h` is no longer needed when using the event queue, right?

Reply:
True, I'll remote it.